src/tracks/labeltrack/ui/LabelTrackView.h

```diff
@@ -58,16 +63,23 @@ class AUDACITY_DLL_API LabelTrackView final : public CommonTrackView
    static const LabelTrackView &Get( const LabelTrack& );
 
    bool DoCaptureKey( AudacityProject &project, wxKeyEvent &event );
-   bool DoKeyDown(
+   void DoKeyDown(
       AudacityProject &project, NotifyingSelectedRegion &sel, wxKeyEvent & event);
-   bool DoChar(
+   void DoChar(
       AudacityProject &project, NotifyingSelectedRegion &sel, wxKeyEvent & event);
 
    //This returns the index of the label we just added.
    int AddLabel(const SelectedRegion &region,
       const wxString &title = {},
       int restoreFocus = -1);
 
+   virtual void OnTextEditFinished(AudacityProject* project, const wxString& text);
+   virtual void OnTextEditCancelled(AudacityProject* project);
+   virtual void OnTextModified(AudacityProject* project, const wxString& text);
+   virtual void OnTextContextMenu(AudacityProject* project, const wxPoint& position);
```

Comment:
Please add `override` to the four declarations above, to fix compiler warnings in clang